src/Microsoft.OData.ModelBuilder/Vocabularies/Capabilities/DeleteRestrictionsBuilder.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.OData.ModelBuilder.Vocabularies.Capabilities
+{
+    /// <summary>
+    /// Allows building restrictions on delete operations;
+    /// </summary>
+    public class DeleteRestrictionsBuilder : VocabularyBuilder<DeleteRestrictionsType>
+    {
+        /// <summary>
+        /// Initializes a new <see cref="DeleteRestrictionsBuilder"/>
+        /// </summary>
+        public DeleteRestrictionsBuilder()
+            : base("Org.OData.Capabilities.V1.DeleteRestrictions", new DeleteRestrictionsType())
+        {
+        }
+
+        /// <summary>
+        /// Entities can be deleted.
+        /// </summary>
+        /// <param name="deletable">Entities can be deleted.</param>
+        /// <returns><see cref="DeleteRestrictionsBuilder" /></returns>
+        public DeleteRestrictionsBuilder IsDeletable(bool deletable)
+        {
+            Data.Deletable = deletable;
+            return this;
+        }
+
+        /// <summary>
+        /// Required permissions. One of the specified sets of scopes is required to perform the delete.
```

Comment:
Is this comment for the entire method?

Reply:
I do not understand the question :-(
As for the comments, there are picked up directly from the specification eg for [DeleteRestrictionType](https://github.com/oasis-tcs/odata-vocabularies/blob/master/vocabularies/Org.OData.Capabilities.V1.md#DeleteRestrictionsType)